libmcount/dynamic.c

```diff
@@ -242,9 +249,36 @@ static int do_dynamic_update(struct symtabs *symtabs, char *patch_funcs,
 	strv_for_each(&funcs, name, j) {
 		bool found = false;
 		bool csu_skip;
+		char *modname, *mod_delimiter;
 		unsigned i, k;
 		struct sym *sym;
 		struct uftrace_pattern patt;
+		struct uftrace_mmap *map;
+		struct mcount_dynamic_info *mdi = NULL;
+
+		mod_delimiter = strchr(name, '@');
+
+		// main module
+		if (mod_delimiter == NULL) {
+			// first of uftrace_mmap is main module always.
```

Comment:
The comment is duplicate.

Reply:
sry, removed

Reply:
removed!

Reply:
Please do not mix `/* */` and `//` style comments. uftrace uses only old-c style `/* */` comment.

Reply:
That's my preference and I want you to follow.  But I'm not too strict about that..

Reply:
That's my preference and I do want you guys follow.  But I'm not too strict about that.